src/utils/misc.ts

```diff
@@ -51,3 +53,19 @@ export function errorFromHeaders(
 
   return undefined;
 }
+
+export function getSendHeaders(options?: SendOptions): Headers {
```

Comment:
this method does not need the whole SendOptions object, only the headers field

Reply:
Fixed